api/middleware/middleware_test.go

```diff
@@ -264,38 +274,3 @@ func TestTokenMiddleware_Success_FromCache(t *testing.T) {
 	session.Get = originalGetCache
 
 }
-
-func TestStoreDatasets(t *testing.T) {
```

Comment:
why did we remove these tests ?

Reply:
Argh! 😱 
I removed them for a bit when I tested removing the dataset functions (because they became so small). Restored them now!